### PR TITLE
[feature] support CentOS (fixes #3)

### LINKED ISSUE #3: the role does not support CentOS
no yum repository.
rpm is available on the official site.

### PR DESCRIPTION
**Pull request by mitsururike**

### CODE REVIEW

File: tasks/install-RedHat.yml [hunk 1]
```diff
@@ -1,6 +1,51 @@
 ---
 
+- name: Check Hshicorp's gpg key is installed
+  command: gpg --list-keys 348FFC4C
+  register: register_vagrant_gpg_key
+  failed_when: register_vagrant_gpg_key.rc not in [0, 2]
+  changed_when: False
+
+- name: Add Hashicorp's gpg key
```
Comment (trombik) [inline tasks/install-RedHat.yml]: http://docs.ansible.com/ansible/rpm_key_module.html

Comment (trombik) [inline tasks/install-RedHat.yml]: with yum module, other tasks can be removed.

http://docs.ansible.com/ansible/yum_module.html
> You can also pass a url or a local path to a rpm file (using state=present).

File: README.md [hunk 2]
```diff
@@ -10,7 +10,12 @@ None
 
 | variable | description | default |
 |----------|-------------|---------|
+| vagrant_version_redhat | Vagrant version in RedHat | {{ __vagrant_version_redhat\|default('') }} |
```
Comment (trombik) [inline README.md]: would you add a URL where you can find a list of available versions?
https://releases.hashicorp.com/vagrant/

File: tasks/install-RedHat.yml (hunk 1 above)
Comment (mitsururike) [inline tasks/install-RedHat.yml]: they are necessary because the rpm is not signed.

**Review by trombik — APPROVED** [review]

**Pull request merged**